marlowe-integration-tests/test/Language/Marlowe/Runtime/Web/StandardContract.hs

```diff
@@ -0,0 +1,201 @@
+  now <- liftIO getCurrentTime
+  let (contract, partyA, partyB) = standardContract partyBAddress now $ secondsToNominalDiffTime 100
+
+  contractCreated@Web.CreateTxBody{contractId} <- postContract
+    partyAWebChangeAddress
+    (Just partyAWebExtraAddresses)
+    (Just partyAWebCollataralUtxos)
+    Web.PostContractsRequest
+      { metadata = mempty
+      , version = Web.V1
+      , roles = Just $ Web.Mint $ Map.singleton "PartyA" $ RoleTokenSimple partyAWebChangeAddress
+      , contract = contract
+      , minUTxODeposit = 2_000_000
+      , tags = mempty
+      }
+
+  createdBlock <- submitContract partyAWallet contractCreated
+
+  pure StandardContractInit
+    { createdBlock
+    , contractCreated
+    , makeInitialDeposit = do
+        initialFundsDeposited <- deposit
+          partyAWallet
+          contractId
+          partyA
+          partyA
+          ada
+          100_000_000
+        initialDepositBlock <- submitTransaction partyAWallet initialFundsDeposited
+
+        pure StandardContractFundsDeposited
+          { initialDepositBlock
+          , initialFundsDeposited
+          , chooseGimmeTheMoney = do
+              gimmeTheMoneyChosen <- choose
+                partyBWallet
+                contractId
+                "Gimme the money"
+                partyB
+                0
+              choiceBlock <- submitTransaction partyBWallet gimmeTheMoneyChosen
+
+              pure StandardContractChoiceMade
+                { choiceBlock
+                , gimmeTheMoneyChosen
+                , sendNotify = do
+                    notified <- notify partyAWallet contractId
+                    notifiedBlock <- submitTransaction partyAWallet notified
+
+                    pure StandardContractNotified
+                      { notifiedBlock
+                      , notified
+                      , makeReturnDeposit = do
+                          returnDeposited <- deposit
+                            partyBWallet
+                            contractId
+                            partyA
+                            partyB
+                            ada
+                            100_000_000
+                          returnDepositBlock <- submitTransaction partyBWallet returnDeposited
+
+                          pure StandardContractClosed
+                            { returnDepositBlock
+                            , returnDeposited
+                            , withdrawPartyAFunds = do
+                                withdrawTxBody <- withdraw partyAWallet contractId "PartyA"
+                                (withdrawTxBody,) <$> submitWithdrawal partyAWallet withdrawTxBody
+                            }
+                      }
+                }
+          }
+    }
+
+standardContract
+  :: PV2.Address
+  -> UTCTime
+  -> NominalDiffTime
+  -> (Contract, Party, Party)
+standardContract partyBAddress startTime timeoutLength = (contract, partyA, partyB)
```

Comment:
Instead of copying this here, could you please import it from `Language.Marlowe.Runtime.Integration.StandardContract`?

Reply:
Thanks @jhbertra done.